packages/frint-model/src/Model.js

```diff
@@ -1,15 +1,44 @@
 import _ from 'lodash';
+import { BehaviorSubject } from 'rxjs';
 
 function Model(attributes) {
   this.attributes = Object.assign({}, attributes);
+  this.$ = null;
 }
 
-Model.prototype.get = function get(key) {
+function getFromAttributes(attributes, key) {
```

Comment:
till now, we have always maintained the API documentation in package's own README whenever we made changes in code. but I agree jsdoc would be a better approach.

would really welcome PRs to speed that process up :)